projects/components/src/public-api.ts

```diff
@@ -261,3 +261,7 @@ export { GreetingLabelComponent } from './greeting-label/greeting-label.componen
 export { MenuDropdownComponent } from './menu-dropdown/menu-dropdown.component';
 export { MenuItemComponent } from './menu-dropdown/menu-item/menu-item.component';
 export { MenuDropdownModule } from './menu-dropdown/menu-dropdown.module';
+
+// Filter builders
+export { NumberFilterBuilder } from './filtering/filter/builder/types/number-filter-builder';
```

Comment:
These shouldn't need to be exported, thats why they're abstracted by the lookup service.